.ignore

```diff
@@ -0,0 +1 @@
+vendor/
```

Comment:
why are we ignoring vendor?

Reply:
it doesn't ignore in git, just in other things like CtrlP or fzf